speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/confscreens/controlconfig.cpp

```diff
@@ -3,30 +3,31 @@
     file        : controlconfig.cpp
     created     : Wed Mar 12 21:20:34 CET 2003
     copyright   : (C) 2003 by Eric Espie
-    email       : eric.espie@torcs.org   
+    email       : eric.espie@torcs.org
     version     : $Id: controlconfig.cpp 6966 2020-04-25 17:33:03Z scttgs0 $
 
  ***************************************************************************/
 
-/***************************************************************************
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
-/** @file   
-    		Human player control configuration menu
-    @author	<a href=mailto:eric.espie@torcs.org>Eric Espie</a>
-    @version	$Id: controlconfig.cpp 6966 2020-04-25 17:33:03Z scttgs0 $
-*/
+ /***************************************************************************
+  *                                                                         *
+  *   This program is free software; you can redistribute it and/or modify  *
+  *   it under the terms of the GNU General Public License as published by  *
+  *   the Free Software Foundation; either version 2 of the License, or     *
+  *   (at your option) any later version.                                   *
+  *                                                                         *
+  ***************************************************************************/
+
+  /** @file
+              Human player control configuration menu
+      @author	<a href=mailto:eric.espie@torcs.org>Eric Espie</a>
+      @version	$Id: controlconfig.cpp 6966 2020-04-25 17:33:03Z scttgs0 $
+  */
```

Comment:
Please do not run the automatic linter on a speed-dreams file.

Because of this I cannot really find which changes were *actually* made, because almost every line his changed due to removal of tabs, moving pointer from variable to type, etc, etc.

Is it possible to revert the linted file and then add your changes to that?